lib/aur-build

```diff
@@ -113,13 +108,21 @@ while true; do
             shift; makechrootpkg_args+=(-d"$1") ;;
         --prefix)
             shift; prefix=$1 ;;
+        -N|--namcap)
+            makechrootpkg_args+=(-n) ;;
+        --checkpkg) # XXX short option
+            makechrootpkg_args+=(-C) ;;
         -T|--temp)
             makechrootpkg_args+=(-T) ;;
+        -U|--user)
+            shift; makechrootpkg_args+=(-U "$1") ;;
         # makepkg options
         -A|--ignorearch|--ignore-arch)
-            makepkg_args+=(--ignorearch) ;;
+            makepkg_args+=(--ignorearch)
+            makechrootpkg_makepkg_args+=(--ignorearch) ;;
         -C|--clean)
             makepkg_args+=(--clean) ;;
+            #makechrootpkg_makepkg_args+=(--clean)
```

Comment:
commented on purpose?

Reply:
Yeah, I didn't know if it had an actual use inside the chroot and left it as a reminder. I can leave out the comment again, though.

Reply:
Removed the comment since aur-build fixes `makechrootpkg -c` anyways.